environments/prod/apps-hmcts-net.yml

```diff
@@ -39,3 +39,23 @@ cname:
   - name: "crime-portal-nle"
     ttl: 300
     record: "crimeportal.apps-nle.hmcts.net"
+
+    # DARTS
+  - name: "darts-portal"
+    ttl: 300
+    record: "sdshmcts-prod-egd0dscwgwh0bpdq.z01.azurefd.net"
+    shutter: false
+  - name: "darts-portal-staging"
+    ttl: 300
+    record: "sdshmcts-stg-abfwhrf8g0btcqhe.z01.azurefd.net"
```

Comment:
@cakeben @scott-robertson1  should the record be `hmcts-live-int.azurefd.net`?

Also, I assume the name is the DNS entry prefix, so we need these to be `darts.staging` for darts.staging.apps.hmcts.net`?